setup.py

```diff
@@ -117,9 +117,17 @@ def get_package_data():
 				data[""].append(os.path.join(root, filename)[11:])
 	return data
 
+def get_version():
+	version_cmd = ['git', 'describe', '--tags', '|' ,'cut', '-c', '1-5']
```

Comment:
doesn't work if downloaded using release zip/tar.gz

Reply:
Yes unfortunately. I guess we're going to have to default to manual versioning for now.